### PR TITLE
aldryn config: remove protected, add excluded

### PR DESCRIPTION
**Pull request by Chive**

### CODE REVIEW

File: boilerplate.json [hunk 1]
```diff
@@ -8,46 +8,13 @@
...
-        "bower.json",
-        "browserslist",
-        "gulpfile.js",
-        "package.json"
+    "excluded": [
+        "docs",
+        "node_modules",
+        "tests",
+        "tools/server",
+        "*.travis.yml",
```
Comment (vxsx) [inline boilerplate.json]: will it expand correctly?

**Comment by vxsx** [comment]
otherwise 👍

File: boilerplate.json [hunk 2]
```diff
@@ -8,46 +8,13 @@
...
-        ".eslintrc.json",
-        "bower.json",
-        "browserslist",
-        "gulpfile.js",
-        "package.json"
+    "excluded": [
+        "docs",
+        "node_modules",
+        "tests",
+        "tools/server",
```
Comment (FinalAngel) [inline boilerplate.json]: also add tools/release.sh

**Comment by FinalAngel** [comment]
also add release.sh and its 👍

File: boilerplate.json (hunk 1 above)
Comment (Chive) [inline boilerplate.json]: not sure why i added an asterisk there.. thanks

**Pull request merged**